Sources/Widgets/Representations/ArrowHandleRepresentation/Constants.js

```diff
@@ -0,0 +1,19 @@
+import vtkArrow2DSource from 'vtk.js/Sources/Filters/Sources/Arrow2DSource';
+import vtkConeSource from 'vtk.js/Sources/Filters/Sources/ConeSource';
+import vtkSphereSource from 'vtk.js/Sources/Filters/Sources/SphereSource';
+import vtkCircleSource from 'vtk.js/Sources/Filters/Sources/CircleSource';
+import vtkCubeSource from 'vtk.js/Sources/Filters/Sources/CubeSource';
+import vtkViewFinderSource from 'vtk.js/Sources/Filters/Sources/ViewFinderSource';
+
+export const ShapeSource = {
+  ARROWHEAD: vtkArrow2DSource,
+  CONE: vtkConeSource,
+  SPHERE: vtkSphereSource,
+  CUBE: vtkCubeSource,
+  CIRCLE: vtkCircleSource,
+  VIEWFINDER: vtkViewFinderSource,
+};
```

Comment:
Sorry, I was not clear enough, I did not mean to create an enum for the shape source.
I meant to create an enum for the vtkArrow2DSource different shapes.

Reply:
Please remove this shape mapping. The includes do not make it a "constant".